test/fixtures/cdb_crankshaft/cdb_voronoi.sql

```diff
@@ -0,0 +1,15 @@
+CREATE SCHEMA IF NOT EXISTS cdb_crankshaft;
```

Comment:
I did it in: https://github.com/CartoDB/camshaft/blob/91b2527fa28bec4bf8daa449045708a8e620672a/test/fixtures/cdb_crankshaft/schema.sql

just waiting to merge everything